docs/release-notes/3.0.0_Beta.md

```diff
@@ -0,0 +1,201 @@
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations—regardless of whether your databases are located on-premises or in the cloud.
+
+## Release highlights
+
+### Encryption of sensitive data
+
+To strengthen the security of your monitoring setup, all sensitive information stored in the PMM Server database, including usernames, passwords, AWS keys, Azure credentials, and TLS/SSL certificates, is now encrypted.
+
+By default, an encryption key is automatically generated and stored at `/srv/pmm-encryption.key`. You also have the option to specify a custom path for the encryption key using the new environment variable `PMM_ENCRYPTION_KEY_PATH`.
+
+When upgrading to PMM 3, any existing unencrypted PMM 2 data will be encrypted automatically.
+
+### Grafana Angular support discontinuation
+
+Grafana will discontinue support for Angular starting with version 12, expected in 2025. This affects numerous panels and plugins, including but not limited to Graph and Table panels. 
+
+We have already migrated many plugins to newer technologies and are actively working on the remaining components to ensure continued functionality. We recommend that you review all plugins in your dashboards and begin planning transitions to newer panel types where necessary.
+
+For a comprehensive list of affected plugins and guidance on migration, see [Grafana's official documentation](https://grafana.com/docs/grafana/latest/developers/angular_deprecation/angular-plugins/) on Angular deprecation and plugin migration.
+
+We will provide regular updates on our migration progress in future releases to help you prepare for this change and modernize your dashboards.
+
+## New default password for AWS Marketplace instances
+
+PMM Server instances launched via AWS Marketplace now use the EC2 instance ID as the default password for the '*admin*' user.
+
+This change stems from our recent transition to Docker-based deployment, ensuring secure and consistent initial access across all new AWS Marketplace instances.
+
+Only new PMM instances launched via AWS Marketplace after this update. For existing instances, continue to use 'admin' as the default password.
+
+Make sure to change the default password immediately after first login.
+
+For more information, see [Install PMM Server on AWS Marketplace](../install-pmm/install-pmm-server/aws/aws.md)
+
+
+
+### Breaking API changes
+
+#### Removed database ID prefixes
+
+We have removed prefixes from database record identifiers (IDs) in PMM to improve API compatibility with REST and to simplify ID handling.
+
+This change affects various components, including nodes, agents, services, and backup-related entities. As a result:
```

Comment:
@catalinaadam On a second read, I believe we need to provide an example of an ID before and after, so users get a better sense of the change, i.e. before - `/agent_id/7cae8a44-8210-4f00-a679-764fa8303ee8`, after - `7cae8a44-8210-4f00-a679-764fa8303ee8`. 

Can we do that?

Reply:
@ademidoff, yep, I've updated the desc for this entry